sqlengine/mysql_engine.go

```diff
@@ -34,6 +34,11 @@ func (d *MySQLEngine) Open(address string, port int64, dbname string, username s
 	return nil
 }
 
+func (d *MySQLEngine) Ping() error {
+	panic("Not implemented")
```

Comment:
Because we should never come here.
We would implement it when we look at MySQL.